tests/ecosystem/deployment/test_deployment.py

```diff
@@ -162,3 +163,13 @@ def ocs_install_verification():
         item['metadata']['name'] for item in storage_classes['items']
     }
     assert required_storage_classes.issubset(storage_class_names)
+
+    # Verify OSD's are distributed
+    log.info("Verifying OSD's are distributed evenly across worker nodes")
+    ocp_pod_obj = ocp.OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    osds = ocp_pod_obj.get(selector='app=rook-ceph-osd')['items']
```

Comment:
Selector is defined already in ocs_ci/ocs/constants.py OSD_APP_LABEL = "app=rook-ceph-osd"

Reply:
Done